org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -231,11 +231,8 @@ object SDTTestUtils extends HasLogger {
   def createSourcePackage(name: String)(project: IScalaProject): IPackageFragment =
     project.javaProject.getPackageFragmentRoot(project.underlying.getFolder("/src")).createPackageFragment(name, true, null)
 
-  def createCompilationUnit(pack: IPackageFragment, name: String, sourceCode: String): ICompilationUnit = {
-    val monitor = new BlockingProgressMonitor
-    val cu = pack.createCompilationUnit(name, sourceCode, false, monitor)
-    monitor.waitUntilDone()
-    cu
+  def createCompilationUnit(pack: IPackageFragment, name: String, sourceCode: String, force: Boolean = false): ICompilationUnit = {
```

Comment:
I'm wondering if changing the signature breaks any clients (scala-worksheet, search or play2). They reuse some of these test utility classes.